src/util/int64.js

```diff
@@ -0,0 +1,45 @@
+function toBuffer (str) {
+  let buf = Buffer.from([ 0, 0, 0, 0, 0, 0, 0, 0 ])
+  let cursor = 0
+  let negative = false
+  if (str[0] === '-') {
+    negative = true
+    buf = Buffer.from([ 255, 255, 255, 255, 255, 255, 255, 255 ])
```

Comment:
I think this is doing a one's complement rather than two's complement, because `-0` and `0` will yield two different buffers. You need to add one after the `for (let i = 0; i < bytes.length; i++)` loop when the number is negative in order to make it the right format. But I think it would be best if we found a module to do this instead of implementing it here